src/test/osdc/object_cacher_stress.cc

```diff
@@ -47,7 +47,7 @@ class C_Count : public Context {
     : m_op(op), m_outstanding(outstanding) {}
   void finish(int r) override {
     m_op->done++;
-    assert(m_outstanding > 0);
+    assert(*m_outstanding > 0);
     *m_outstanding--;
```

Comment:
@chardan The fix done by @wjwithagen is good and it eliminates the comparison warning. But It again gives the below warning for me. So when I reviewed the code, I have noticed that m_outstanding is just used for asserting and the decremented value is not used at all. So is it implemented just for asserting ? Is it an unwanted member variable that we can remove ? These are the doubts that I have. Please clarify the purpose of implementing it. Thanks.

```
ceph/src/test/osdc/object_cacher_stress.cc: In member function ‘virtual void C_Count::finish(int)’:
ceph/src/test/osdc/object_cacher_stress.cc:51:5: warning: value computed is not used [-Wunused-value]
*m_outstanding--;
```